code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -19,6 +19,20 @@
 	STOP_PROCESSING(SSobj, src)
 	return ..()
 
+/obj/item/device/radio/intercom/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/weapon/wrench))

```

Comment:
Yeah, I was afraid of the whole tiding thing. I'll probably add another step to prevent it.